### PR TITLE
Feature: youtube player clone page

### PR DESCRIPTION
**Pull request by StarSheriff2**
The goal of this project was to clone the Youtube player page using HTML and CSS only

- semantic HTML tags to separate sections
- mostly used flexbox for positioning and centering
- all icons are separate in the image folder
- thumbnail pictures are found in the thumbnail subfolder
- external links and buttons don't work

### CODE REVIEW

**Review by kevcap — CHANGES_REQUESTED** [review]
# [STATUS] CHANGES REQUIRED♻️

#### Hey team,  your project is looking really good, however, to accomplish as the actual page, you required to do these changes, listed below:
 
_Note_: That changes marked as [OPTIONAL] are not required to approve your project, but I strongly recommend that you guys go for that extra mile, to get more knowledge and have a better project to add at your portfolio.
 
 
## [REQUIRED CHANGES]
- [x] Your page is broken, the video is too big, please fix it.
- [x] Your page has a horizontal bar, please remove it (fixing the size of the elements might work)
image

## [OPTIONAL CHANGES]
**⚠These are mandatory changes, but since It wasn't pointed out earlier, I'll mark them as [OPTIONAL]:**
- [ ] Kindly remember to remove all the unnecessarily repeated CSS code. Keeping your code DRY is an excellent practice that will help you a lot and make your code more professional. (Check this link form better understanding [Good and Bad CSS Practices for Beginners](https://speckyboy.com/good-bad-css-practices/)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)
Take a look at this example:
image
Your HTML will be something like this: 
image
Apply the same logic above in **ALL** those repeated rules listed below:
image

## Please make the required changes and resubmit for a new code review once you are ready💪

## As always, I wish you all the very best and see you both at the top🚀


👨🏻‍💻 **TSE: Kevin Alves**

  

📫 Have any question? Better call Kevin:

<a  href="mailto:kevinalves.js@gmail.com"> gmail</a> [Twitter Follow](https://twitter.com/intent/follow?screen_name=kevinnnmn) [](https://microverse-students.slack.com/team/UT86D3AP4)

<sub>*We're here to help you, to ensure that you're using the best practices and getting you ready for the job market. In real-world, you'll have a lot of changes requests for your real clients, it's good to get used to it.</sub>

File: css/styles.css [hunk 1]
```diff
@@ -0,0 +1,366 @@
...
+  margin: 0 10px;
+}
+
+.footer-icon-img {
+  height: 20px;
+  width: auto;
+  padding-right: 10px;
+}
+
+.footer-links a, .footer-help a {
```
Comment (teekaytech) [inline css/styles.css]: When you are applying a block to multiple selectors (as in line 354), always break individual selector to another line. Something like this: 
```
.footer-links a,
.footer-help a {
--- rules here ---
}
```

File: index.html [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+        <div class="title-btns flex-horizontal">
+          <img class="title-icons" src="images/plus.png" alt="plus icon"><span>Add To</span>
+          <img class="title-icons" src="images/share.png" alt="share icon"><span>Share</span>
+          <img class="title-icons" src="images/ellipsis.png" alt="ellipsis icon"> <span>More</span>
+        </div>
+        <div class="like-btns flex-horizontal">
+          <img class="title-icons" src="images/like.png" alt="like icon"><span class="like-num">888</span>
+          <img class="title-icons" src="images/dislike.png" alt="dislike icon"><span>12</span>
+        </div>
+      </footer>
```
Comment (teekaytech) [inline index.html]: Nesting this section inside a `<footer>` tag is not necessary. Semantically, the `<footer>` tag has its meaning and it should only be used for the footer section of your page. Kindly replace the footer tag with another tag, probably with a `<div>` or `<section>`

**Review by teekaytech — COMMENTED** [review]
# Status: Changes Required :recycle: :vertical_traffic_light: 

## Hi team :wave: 
### Nice work on making the changes raised by the previous TSE :muscle: . 
- Your project is looking great. :+1: 
- You have a descriptive readme file. :100: 
- Your codebase is well structured. :+1: 

**However, there are some little changes that need to be made before the project can be approved. Kindly see make the changes listed below and request another review afterward.**

**[REQUIRED]**

- [x] Right now, you have two icons to the leftmost part of your navbar. Although that is what we have on the original page, but the requirement says you should include one more icon to make it three. You can look at the current youtube page and add any icon that is missing to satisfy this requirement.

A

- [x] Check my in-line comments below :point_down:  and make the changes as required.

**[OPTIONAL]**
_The changes listed here are not required. You can skip them and your project will be approved. However, implementing them makes your project better._
- [ ] You are encouraged to implement the comments section of the page.

b

### You're almost there, this is already a great project. Kindly make the finishing touches and request for another review.

* * *

**_You can reach out to me on twitter (@ola_lere), Github (@teekaytech) or slack (@taofeekolalere )_**


Happy Coding :man_technologist:

**Review by jubaan — CHANGES_REQUESTED** [review]
# Status: Changes Required

## :wave: :wave: Hello team! I've seen you applied the suggested changes that's Awesome!

[Mandatory]

- [ ] **search button** separated from the search bar. It's strange as in the **preview image** you have in the **README** it's the code had already solved this issue.

**Actual:**
image

**README Image**
image

## Keep up the great work :muscle: and Happy Coding :smiley: :computer:!!!!



## [NOTES]

It got my attention that you closed the previous pull request. I think this was related to the branch name optional change. If this was the case ```git``` has options to rename local and remote branches. I'll leave you a link that might help in the future. 

- [git branch -m](https://www.w3docs.com/snippets/git/how-to-rename-git-local-and-remote-branches.html)

**Review by kevcap — CHANGES_REQUESTED** [review]
# [STATUS] CHANGES REQUIRED♻️

#### Hey team,  your project is looking really good, however, to accomplish as the actual page, you required to do these changes, listed below:
 
 
## [REQUIRED CHANGES]
- [x] Your search button is yet separated from your search input.
- [x] Your video is still too big and doesn't have left margins.
- Yours
image

- Original
image

## [OPTIONAL CHANGES]
- [x] Improve the layout to make it more similar to the original page.

## Please make the required changes and resubmit for a new code review once you are ready💪

## As always, I wish you all the very best and see you both at the top🚀


👨🏻‍💻 **TSE: Kevin Alves**

  

📫 Have any question? Better call Kevin:

<a  href="mailto:kevinalves.js@gmail.com"> gmail</a> [Twitter Follow](https://twitter.com/intent/follow?screen_name=kevinnnmn) [](https://microverse-students.slack.com/team/UT86D3AP4)

<sub>*We're here to help you, to ensure that you're using the best practices and getting you ready for the job market. In real-world, you'll have a lot of changes requests for your real clients, it's good to get used to it.</sub>

**Comment by StarSheriff2** [comment]
Note: This youtube page was cloned from an old Youtube page from 2014 per project requirements defined in the Microverse pathright site: [](http://archive.vn/Bss88). We tried to make it look as close as possible to that page, and not the current Youtube version.

**Review by jubaan — APPROVED** [review]
# Status: Approved :tada: :tada: :tada:

## Congrats! :smiley: you've worked on all the required changes. You may now merge with master :balloon: :balloon: :clap: :clap: :clap:

**Pull request merged**